addons/controllers/antreaconfig_controller_test.go

```diff
@@ -19,7 +19,7 @@ import (
 	cniv1alpha1 "github.com/vmware-tanzu/tanzu-framework/apis/cni/v1alpha1"
 )
 
-var _ = Describe("AntreaConfig Reconciler", func() {
+var _ = FDescribe("AntreaConfig Reconciler", func() {
```

Comment:
blocking: we might want to change back to `Describe` before merge.

Reply:
I don't plan to check this in. This is to only facilitate testing for time being